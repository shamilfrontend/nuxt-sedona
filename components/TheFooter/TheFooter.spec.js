import { shallowMount } from '@vue/test-utils'
import Component from './index'

describe('TheFooter', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {
      mocks: {
        $route: {
          path: '/'
        }
      }
    })
  })

  test('mounts properly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
