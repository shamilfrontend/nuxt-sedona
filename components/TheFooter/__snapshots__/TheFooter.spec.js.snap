// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter should match snapshot 1`] = `
<footer class="page-footer page-footer_is-home-page"><span class="visit">#Visitsedona</span>
  <ul class="social-list">
    <li class="social social-tw"><a href="https://twitter.com/" title="Twitter" target="_blank" class="social-button"><svg height="15" width="17" viewBox="0 0 17 15" xmlns="http://www.w3.org/2000/svg">
          <path d="M10.95.144c1.685-.496 2.984.27 3.577 1.179.673-.231 1.331-.481 2.011-.708a2.345 2.345 0 0 1-.857 1.768c.685.17 1.304-.491 1.304-.491-.169 1-1.006 1.788-1.563 2.024-.231 6.75-3.175 11.217-10.077 11.082h-.446c-.41 0-4.164-.46-4.898-1.887 2.271.196 3.893-.422 4.693-1.177-.96-.3-2.679-.477-2.979-2.95.349.106.564.228 1.19.119C1.705 8.247.374 7.53.448 5.33c.285.328 1.067.536 1.34.472C1.085 5.561-.182 2.442.894.85c1.818 1.854 3.735 3.606 7.152 3.773C8.254 2.33 9.183.793 10.95.144z" fill="#FFF"></path>
        </svg></a></li>
    <li class="social social-fb"><a href="https://www.facebook.com/" title="Facebook" target="_blank" class="social-button"><svg height="22" width="12" viewBox="0 0 12 22" xmlns="http://www.w3.org/2000/svg">
          <path d="M12 4V0H8a4 4 0 0 0-4 4v4H0v4h4v10h4V12h4V8H8V4h4z" fill="#FFF"></path>
        </svg></a></li>
    <li class="social social-vk"><a href="https://vk.com/" title="YouTube" target="_blank" class="social-button"><svg height="16" width="20" viewBox="0 0 20 16" xmlns="http://www.w3.org/2000/svg">
          <path d="M17 0H3C1.35 0 0 1.35 0 3v10c0 1.65 1.35 3 3 3h14c1.65 0 3-1.35 3-3V3c0-1.65-1.35-3-3-3zM6.027 11.998V4.002L15.014 8l-8.987 3.998z" fill="#FFF"></path>
        </svg></a></li>
  </ul>
  <div class="footer-copyright"><a href="http://alisultanov.ru/" class="footer-copyright-link"><span class="website">Website by</span> <span>SHAMILFRONTEND</span></a></div>
</footer>
`;
